README.md

```diff
@@ -52,15 +52,82 @@ Technology is a powerful force in our society. Data, software, and communication
 - [Zerocoin paper](https://zerocoin.org/media/pdf/ZerocoinOakland.pdf)
 - [Zerocash paper](http://zerocash-project.org/media/pdf/zerocash-oakland2014.pdf)
 
-## Courses
+### Courses
```

Comment:
```suggestion
## Courses
```
As these courses aren't about distributed systems they should be at a higher level of heading